Manito/Manito/Screens/Detail-Wait/DetailEditViewController.swift

```diff
@@ -0,0 +1,207 @@
+        return button
+    }()
+    private let topIndicator: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white.withAlphaComponent(0.8)
+        view.layer.cornerRadius = 1.5
+        return view
+    }()
+    private lazy var changeButton: UIButton = {
+        let button = UIButton(type: .system)
+        let buttonAction = UIAction { _ in
+            self.dismiss(animated: true)
+        }
+        button.setTitle("변경", for: .normal)
+        button.setTitleColor(.subBlue, for: .normal)
+        button.titleLabel?.font = .font(.regular, ofSize: 16)
+        button.addAction(buttonAction, for: .touchUpInside)
+        return button
+    }()
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "방 정보 수정"
+        label.font = .font(.regular, ofSize: 16)
+        return label
+    }()
+    private let startSettingLabel: UILabel = {
+        let label = UILabel()
+        label.text = "진행기간 설정"
+        label.font = .font(.regular, ofSize: 16)
+        label.textColor = .white
+        return label
+    }()
+    private let calendar = CalendarView()
+    private let tipLabel: UILabel = {
+        let label = UILabel()
+        label.text = "최대 7일까지 설정할 수 있어요 !"
+        label.textColor = .grey005
+        label.font = .font(.regular, ofSize: 14)
+        return label
+    }()
+    private let setMemberLabel: UILabel = {
+        let label = UILabel()
+        label.text = "인원 설정"
+        label.font = .font(.regular, ofSize: 18)
+        label.textColor = .white
+        return label
+    }()
+    private let minMemberLabel: UILabel = {
+        let label = UILabel()
+        label.text = "5인"
+        label.font = .font(.regular, ofSize: 16)
+        label.textColor = .white
+        return label
+    }()
+    private let maxMemberLabel: UILabel = {
+        let label = UILabel()
+        label.text = "15인"
+        label.font = .font(.regular, ofSize: 16)
+        label.textColor = .white
+        return label
+    }()
+    private lazy var memberSlider: UISlider = {
+        let slider = UISlider()
+        slider.minimumValue = 5
+        slider.maximumValue = 15
+        slider.maximumTrackTintColor = .grey006
+        slider.minimumTrackTintColor = .darkRed
+        slider.value = Float(memberCount)
+        slider.isContinuous = true
+        slider.addTarget(self, action: #selector(changeMemberCount(sender:)), for: .valueChanged)
+        return slider
+    }()
+    private lazy var memberCountLabel: UILabel = {
+        let label = UILabel()
+        label.text = "\(memberCount)인"
+        label.font = .font(.regular, ofSize: 24)
+        label.textColor = .white
+        return label
+    }()
+
+    // MARK: - life cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    override func configUI() {
+        self.navigationController?.isNavigationBarHidden = true
+        view.backgroundColor = .darkGrey002
+    }
```

Comment:
p1;

```suggestion
    override func configUI() {
        super.configUI()
        self.navigationController?.isNavigationBarHidden = true
    }
```

이런식으로 변경하면 제가 BaseViewController 내부에 설정해둔 backgroundColor를 사용할 수 있습니다!

Reply:
대박이네요... 
듀나 멋져 ^_____^